app/components/channel_drawer/teams_list/index.js

```diff
@@ -33,4 +35,8 @@ function mapDispatchToProps(dispatch) {
     };
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(TeamsList);
+function areStatesEqual(next, prev) {
```

Comment:
Like most of the others, I don't think this is needed. It'll also break if the theme or locale change